Lib9c/Action/RankingBattle4.cs

```diff
@@ -0,0 +1,199 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Numerics;
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Action;
+using Libplanet.Assets;
+using Nekoyume.Battle;
+using Nekoyume.Model.BattleStatus;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using Serilog;
+
+namespace Nekoyume.Action
+{
+    [Serializable]
+    [ActionType("ranking_battle")]
```

Comment:
`RankingBattle4` ActionType is `ranking_battle`. `RankingBattle` ActionType is `ranking_battle4`.
Actiontypes seem to have been reversed. Is there any reason?